server.js

```diff
@@ -63,6 +63,6 @@ function normalizePort(val) {
 // Set up the port
 let port = normalizePort(process.env.PORT || '3000');
 
-server.listen(port, () => {
+server.listen(3000, () => {
```

Comment:
Needs to be `port` so it works on heroku

Reply:
Changed